### PR TITLE
build: rename and bump up dependencies

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would:
- rename the module name from github.com/line/finschia to github.com/Finschia/finschia
- rename the following dependencies (and update scripts when necessary)
  - github.com/line/lbm-sdk => github.com/Finschia/finschia-sdk
  - github.com/line/ostracon => github.com/Finschia/ostracon
  - github.com/line/ibc-go => github.com/FInschia/ibc-go
  - github.com/line/wasmd => github.com/Finschia/wasmd
- change the ci runners from self-hosted to ubuntu-latest
- bump up Finschia/finschia-sdk
  - Finschia/finschia-sdk#970
  - Finschia/finschia-sdk#982
  - Finschia/finschia-sdk#980

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/finschia/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/finschia/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
Please check below files and change the link from `line/lbm` or `line/finschia`, etc.
* .releaserc
* CHANGELOG.md
* CONTRIBUTIONG.md
* README.md
* RELEASE_NOTE.md

**Comment by 0Tech** [comment]
> Please check below files and change the link from `line/lbm` or `line/finschia`, etc.
> 
>     * .releaserc
> 
>     * CHANGELOG.md
> 
>     * CONTRIBUTIONG.md
> 
>     * README.md
> 
>     * RELEASE_NOTE.md

I thought they would be addressed by another PR. Should I update them right now?

**Comment by zemyblue** [comment]
> > Please check below files and change the link from `line/lbm` or `line/finschia`, etc.
> > ```
> > * .releaserc
> > 
> > * CHANGELOG.md
> > 
> > * CONTRIBUTIONG.md
> > 
> > * README.md
> > 
> > * RELEASE_NOTE.md
> > ```
> 
> I thought they would be addressed by another PR. Should I update them right now?

If you want, please add another PR.

**Review by zemyblue — APPROVED** [review]

**Review by zemyblue — COMMENTED** [review]
Please check `.goreleaser.yml`

https://github.com/Finschia/finschia/blob/fad698791233d5988235e79fcaf907f171ad1711/.goreleaser.yml#L18-L22

**Comment by 0Tech** [comment]
There are several "line" related things remaining:
- docs/delegators/delegator-guide-cli.md
  - the Line Blockchain Lab Team
- docs/DOCS_README.md
  - https://docs-blockchain.line.biz (I found it working)

**Comment by 0Tech** [comment]
Also, the pushing repository name would be updated to `finschia/finschianode`. I found the name in many targets in Makefile.

File: cli_test/test_helpers.go [hunk 1]
```diff
@@ -19,52 +19,52 @@ import (
...
+	ostcfg "github.com/Finschia/ostracon/config"
+	"github.com/Finschia/ostracon/libs/log"
+	osthttp "github.com/Finschia/ostracon/rpc/client/http"
+	ostctypes "github.com/Finschia/ostracon/rpc/core/types"
+	osttypes "github.com/Finschia/ostracon/types"
+	wasmtypes "github.com/Finschia/wasmd/x/wasm/types"
+
+	"github.com/Finschia/finschia/app"
+	lbmcmd "github.com/Finschia/finschia/cmd/fnsad/cmd"
+	lbmtypes "github.com/Finschia/finschia/types"
```
Comment (zemyblue) [inline cli_test/test_helpers.go:67]: How about changing from `lbmcmd`, `lbmtypes` to `fnsacmd`, `fnsatypes`?

File: RELEASE_CHANGELOG.md [hunk 2]
```diff
@@ -134,17 +134,17 @@ This version based on lbm-sdk v0.46.0-rc2
...
 * (sdk) Enable signature verification cache
 * (ostracon) Apply asynchronous receiving reactor
-* (sdk) [\#21](https://github.com/line/lbm/pull/21) Use lbm-sdk v0.43.0
+* (sdk) [\#21](https://github.com/Finschia/finschia/pull/21) Use lbm-sdk v0.43.0
 
 ### Bug Fixes
 
 ### Breaking Changes
 * (sdk) (auth) [\#16](https://github.com/line/lfb/pull/16) Introduce sig block height for the new replay protection
 * (ostracon/sdk) [\#26](https://github.com/line/lfb/pull/26) Use vrf-based consensus, address string treatment
```
Comment (zemyblue) [inline RELEASE_CHANGELOG.md:143]: Please change from `line/lfb` to `Finschia/lbf`.

File: docs/finschia-tutorials/installation.md [hunk 3]
```diff
@@ -27,17 +27,17 @@ source ~/.bash_profile
 ## Install the binaries
```
Comment (zemyblue) [inline docs/finschia-tutorials/installation.md:27]: Please change `LBM SDK` of above 24line.

**Review by zemyblue — APPROVED** [review]

**Review by dudong2 — APPROVED** [review]

**Pull request merged**